### PR TITLE
README: Add our apt repository as a way to install kaidan

### PR DESCRIPTION
**Pull request by JBBgameich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,22 @@ Kaidan is *not* finished yet, so don't expect it working as well as a finished c
...
+
+ * `sudo apt-add-repository "deb https://raw.githubusercontent.com/kaidanim/packages/master sid main"`
+ * `sudo apt-key adv --keyserver hkp://pgp.mit.edu --recv-key "210EB0BCA70439F0"`
+ * `sudo apt-get update`
+
+Then finally install Kaidan
+
+ * `sudo apt-get install kaidan`
+
+### Compile yourself
```
Comment (LNJ2) [inline README.md]: Maybe better `Compile it yourself`

File: README.md [hunk 2]
```diff
@@ -25,6 +25,22 @@ Kaidan is *not* finished yet, so don't expect it working as well as a finished c
...
+
+Note: We can not guarantee that the packages available from the repository is always up to date.
+
+Add our apt repository too your system
+
+ * `sudo apt-add-repository "deb https://raw.githubusercontent.com/kaidanim/packages/master sid main"`
+ * `sudo apt-key adv --keyserver hkp://pgp.mit.edu --recv-key "210EB0BCA70439F0"`
+ * `sudo apt-get update`
+
+Then finally install Kaidan
```
Comment (LNJ2) [inline README.md]: not finally

File: README.md [hunk 3]
```diff
@@ -25,6 +25,22 @@ Kaidan is *not* finished yet, so don't expect it working as well as a finished c
 
 ## Installation
 
+### Debian sid based systems
+
+Note: We can not guarantee that the packages available from the repository is always up to date.
+
+Add our apt repository too your system
```
Comment (LNJ2) [inline README.md]: to not too

Comment (JBBgameich) [inline README.md]: shiit ...

File: README.md (hunk 2 above)
Comment (JBBgameich) [inline README.md]: why not? It's the last point in the list

Comment (LNJ2) [inline README.md]: but it's the second point, so the way to it wasn't so long

Comment (JBBgameich) [inline README.md]: that's true :D

File: README.md [hunk 4]
```diff
@@ -25,6 +25,22 @@ Kaidan is *not* finished yet, so don't expect it working as well as a finished c
 
 ## Installation
 
+### Debian sid based systems
+
+Note: We can not guarantee that the packages available from the repository is always up to date.
```
Comment (LNJ2) [inline README.md]: sry but: you have to use `are` instead of `is`

Comment (JBBgameich) [inline README.md]: so true ...

**Pull request merged**